src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.label--big {
+  position: relative;
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  margin: 20px 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  height: 70px;
+  background-color: #fff;
+}
+
+.label--big:hover {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.label--big:focus {
+  box-sizing: border-box;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.label--small {
+  display: block;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  border-radius: 4px;
+  margin: 20px 0;
+  height: 42px;
+}
+
+.label--small:hover {
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.label--small:focus {
+  box-sizing: border-box;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input--big {
+  outline: none;
+  border: none;
+  font-size: 16px;
+  height: 22px;
+  width: 129px;
+  padding: 0;
+  /*margin: 23px 0 23px 17px;*/
+  position: relative;
+  top: -4px;
+  right: 62px;
+  left: 14px;
+  font-weight: 800;
+}
+
+.input--small {
+  outline: none;
+  border: none;
+  font-size: 14px;
+  position: relative;
+  height: 20px;
+  width: 167.29px;
+  padding: 0;
+  /*margin: 10px 0 10px 9px;*/
+  top: -1px;
+  left: 6.5px;
+}
+
+.img--big {
+  position: relative;
+  width: 19px;
+  height: 19px;
+  padding-left: 25px;
+  padding-top: 24px;
+  margin: 0;
+}
+
+.img--small {
+  position: relative;
+  width: 11px;
+  height: 11px;
+  padding-left: 12px;
+  padding-top: 14px;
+  margin: 0;
+}
+
+.input--big::placeholder {
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.input--small::placeholder {
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 300;
+}
```

Comment:
You duplicate a lot of styles for small and big inputs. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point